### PR TITLE
facet: fix subjects fiction

### PR DESCRIPTION
**Pull request by rerowep**
## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

### CODE REVIEW

**Review by zannkukai — APPROVED** [review]

**Review by PascalRepond — APPROVED** [review]
This is OK as a quick-fix for v1.17.0 but the fiction/non-fiction mechanism will be adapted later as discussed. Fiction should not be a modifier for terms but for documents. I will open an issue.

**Review by Garfield-fr — APPROVED** [review]

**Review by jma — APPROVED** [review]
Tests are missing.

File: rero_ils/config.py [hunk 1]
```diff
@@ -1707,14 +1707,16 @@ def _(x):
                 'facet_subject',
                 must=[{
                     'name_or_query': 'terms',
-                    'genreForm__identifiedBy__value': ['A027757308', 'A021097366']
+                    # TODO: If we have translated concepts we have to addapt this filter.
+                    'facet_genre_form_en': FICTIONS_TERMS
```
Comment (PascalRepond) [inline rero_ils/config.py:1711]: Why not `'genreForm__entity__authorized_access_point': FICTIONS_TERMS` ?

File: rero_ils/config.py [hunk 2]
```diff
@@ -1707,14 +1707,16 @@ def _(x):
...
                 }]
             ),
             _('subject_no_fiction'): and_i18n_term_filter(
                 'facet_subject',
                 must_not=[{
                     'name_or_query': 'terms',
-                    'genreForm__identifiedBy__value': ['A027757308', 'A021097366']}
+                    # TODO: If we have translated concepts we have to addapt this filter.
+                    'facet_genre_form_en': FICTIONS_TERMS}
```
Comment (PascalRepond) [inline rero_ils/config.py:1719]: Why not `'genreForm__entity__authorized_access_point': FICTIONS_TERMS` ?

**Pull request merged**